test/system/252-quadlet.bats

```diff
@@ -1013,4 +1013,81 @@ EOF
     run_podman rmi $(pause_image)
 }
 
+@test "quadlet - image files" {
+    local quadlet_image_unit=image_test_$(random_string).image
+    local quadlet_image_file=$PODMAN_TMPDIR/$quadlet_image_unit
+    cat > $quadlet_image_file <<EOF
+[Image]
+Image=$IMAGE
+EOF
+
+    # Remove the local image to make sure it will be pulled again
+    run_podman image rm $IMAGE
+
+    # Create a directory for all quadlet files to make sure later steps access previous ones
+    local quadlet_tmpdir=$(mktemp -d --tmpdir=$PODMAN_TMPDIR quadlet.XXXXXX)
```

Comment:
Since `$PODMAN_TMPDIR` is already a pseudorandom unique dir, is this really necessary? A maintainer might find this easier to read:
```bash
local quadlet_tmpdir=$PODMAN_TMPDIR/quadlets
```

Reply:
AFAIK `PODMAN_TMPDIR` is random at the suite level and not at the test level. Am I wrong?

Reply:
PODMAN_TMPDIR is newly created for each test

Reply:
Done (along with creating the directory)